platform/persistence/platform-persistence-core-impl/src/test/java/org/codice/ddf/persistence/internal/PersistentStoreImplTest.java

```diff
@@ -111,4 +146,16 @@ public void testGetInvalidQuery() throws Exception {
     assertThat(items.size(), equalTo(1));
     verify(solrClient, never()).query(any(), eq(SolrRequest.METHOD.POST));
   }
+
+  private SolrDocumentList getSolrDocuments(int numDocuments) {
+    final SolrDocumentList docList = new SolrDocumentList();
+
+    for (int i = 0; i < numDocuments; i++) {
+      SolrDocument solrDocument = new SolrDocument();
+      solrDocument.addField("id_txt", String.format("idvalue%d", i + 1));
```

Comment:
✏️ Not usually worth using `String.format` for a single concatenation.

Reply:
I think the string format is fine